packages/react-mutation-mapper/src/index.tsx

```diff
@@ -170,3 +171,4 @@ export { default as DefaultMutationMapperDataStore } from './store/DefaultMutati
 export { default as DefaultMutationMapperFilterApplier } from './store/DefaultMutationMapperFilterApplier';
 export { default as DefaultMutationMapperStore } from './store/DefaultMutationMapperStore';
 export * from './component/oncokb/constants';
+export * from './util/OncoKbUtils';
```

Comment:
Instead of introducing a new export statement we should replace the lines [162-165](https://github.com/cBioPortal/cbioportal-frontend/pull/3826/files#diff-b1ffa3f6b2299ce3783b21c0dd11d9e23ba2c254568da3400726ec2033064bbeR162-R165):

```JavaScript
export {
    defaultOncoKbFilter,
    groupOncoKbIndicatorDataByMutations,
} from './util/OncoKbUtils';
```

Reply:
Good catch!